views/templates/dialogs/dialog-update.html.twig

```diff
@@ -10,25 +10,26 @@
     id="form-confirm-update"
     name="form-confirm-update"
   >
-  {% if not backup_completed %}
-    <p>
-      {{ 'Before starting the update, make sure you have a complete and recent backup of your store (database, files, and images).'|trans({}) }}
-    </p>
-  {% else %}
-    {% set message = 'You are about to launch the update, do you want to continue?'|trans({}) %}
-    {{ parent() }}
-  {% endif %}
+    {% if not backup_completed %}
+      <p>
+        {{ 'Before starting the update, make sure you have a complete and recent backup of your store (database, files, and images).'|trans({}) }}
+      </p>
+    {% else %}
+      {% set message = 'You are about to launch the update, do you want to continue?'|trans({}) %}
+      {{ parent() }}
+    {% endif %}
 {% endblock %}
 
 {% block dialog_extra_content %}
-  {% if not backup_completed %}
-    <div class="dialog__no-backup checkbox">
-      <label>
-        <input type="checkbox" value="" id="dialog-start-update-own-backup" />
-        {{ 'I have made my own backup and can restore it manually if a problem occur.'|trans({}) }}
-      </label>
-    </div>
-  {% endif %}
+    {% if not backup_completed %}
+      <div class="dialog__no-backup checkbox">
+        <label>
+          <input type="checkbox" value="" id="dialog-start-update-own-backup" />
+          {{ 'I have made my own backup and can restore it manually if a problem occur.'|trans({}) }}
+        </label>
+      </div>
+    {% endif %}
+  </form>
```

Comment:
@tblivet need your review here, as the `<form>` is opened in a block then closed in another

Reply:
I'd say that's a bad idea. :-)